.gitignore

```diff
@@ -65,6 +65,7 @@ src/qt/test/moc*.cpp
 *.a
 *.pb.cc
 *.pb.h
+.vscode
```

Comment:
I suggested Simon add it because he was avoiding commiting it, something for his IDE.